public/wp-content/themes/ColoredCow/landing.php

```diff
@@ -5,9 +5,88 @@
     get_header();
 ?>
 
-<?php
-?>
- 
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
+				<h2 class="about">Soiree</h2>
+				<br>
+			  	<p class="lead">ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
+				<hr>
+				<center>
+			  		<div class="col-md-8 col-xs-12">
+						<h5 class="request">Wanna join the party?</h5>
```

Comment:
class="request" looks like an action. If it is just for the text, use a better name like request-question or request-text.

Let's make it a bit more professional. wanna -> Want to